app/config/services/admin.xml

```diff
@@ -309,15 +309,15 @@
         </service>
 
         <service id="app.admin.vote_place_admin" class="AppBundle\Admin\VotePlaceAdmin">
-            <tag name="sonata.admin" manager-type="orm" label="Assésseurs » Bureaux de vote" group="Territoire" />
+            <tag name="sonata.admin" manager-type="orm" label="Assesseurs » Bureaux de vote" group="Territoire" />
```

Comment:
I think we should use the inclusive writing for the `assesseur(e)s` and `assesseur(e)` words everywhere in this PR.
@JulHb had made this request for the form.

Reply:
Ok :)